berserk/clients/tournaments.py

```diff
@@ -22,14 +22,67 @@ def get(self) -> models.CurrentTournaments:
         )
 
     def get_tournament(self, tournament_id: str, page: int = 1):
-        """Get information about a tournament.
+        """Get information about an Arena tournament.
 
-        :param tournament_id
+        :param tournament_id: tournament ID
+        :param page: the page number of the player standings to view
         :return: tournament information
         """
         path = f"/api/tournament/{tournament_id}?page={page}"
         return self._r.get(path, converter=models.Tournament.convert)
 
+    def join_arena(
+        self,
+        tournament_id: str,
+        password: str | None = None,
+        team: str | None = None,
+        should_pair_immediately: bool = False,
+    ) -> None:
+        """Join an Arena tournament. Also, unpauses if you had previously paused the tournament.
+
+        Requires OAuth2 authorization with tournament:write scope.
+
+        :param tournament_id: tournament ID
+        :param password: tournament password or user-specific entry code generated and shared by the organizer
+        :param team: team with which to join the team battle Arena tournament
+        :param should_pair_immediately: if the tournament is started, attempt to pair the user, even if they are not
+            connected to the tournament page. This expires after one minute, to avoid pairing a user who is long gone.
+            You may call "join" again to extend the waiting.
+        """
+        path = f"/api/tournament/{tournament_id}/join"
+        params = {
+            "password": password,
+            "team": team,
+            "pairMeAsap": should_pair_immediately,
+        }
+        self._r.post(path=path, params=params, converter=models.Tournament.convert)
+
+    def get_team_standings(self, tournament_id: str) -> Dict[str, Any]:
```

Comment:
the response is simple enough to be typed: https://lichess.org/api#tag/Arena-tournaments/operation/resultsByTournament please also include a test for it